install_config/persistent_storage/index.adoc

```diff
@@ -31,6 +31,7 @@ Persistent Disk]
 - xref:../../install_config/persistent_storage/persistent_storage_fibre_channel.adoc#install-config-persistent-storage-persistent-storage-fibre-channel[Fibre Channel]
 - xref:../../install_config/persistent_storage/persistent_storage_azure.adoc#install-config-persistent-storage-persistent-storage-azure[Azure Disk]
 - xref:../../install_config/persistent_storage/persistent_storage_azure_file.adoc#install-config-persistent-storage-persistent-storage-azure-file[Azure File]
+- xref:../../install_config/persistent_storage/persistent_storage_flex_volume.adoc#[Flex Volume]
```

Comment:
Update link to be:

xref:../../install_config/persistent_storage/persistent_storage_flex_volume.adoc#install-config-persistent-storage-persistent-storage-flex-volume[Flex Volume]